test/fixtures/v5/exit_rotary/left_destination.json

```diff
@@ -21,7 +21,8 @@
         "it": "Svolta a sinistra verso Destination 1",
         "nl": "Ga linksaf richting Destination 1",
         "pl": "Skręć w lewo w kierunku Destination 1",
-        "pt-BR": "Vire à esquerda sentido Destination 1",
+        "pt-BR": "Exit the traffic circle towards Destination 1",
```

Comment:
Just double checking: the Brazilian Portugese translations are expected to be in English here because the translation is missing, right?

Reply:
Although it’s unfortunate that this string has reverted to English, the previous Portuguese string was suboptimal anyways (“Turn left towards Destination 1”). I suppose it would be possible to pare back this PR to bring back the old string and test fixture, but I’m not sure that should be the bar for adding a separate localization.